lib/user.rb

```diff
@@ -0,0 +1,22 @@
+class User
+# attr_reader :name
+attr_accessor :jokes, :name
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(name, joke)
+    @jokes << joke
+  end
+
+  def joke_by_id(id)
```

Comment:
We were hoping to see you use one of the enumerables that we learned about to iterate over the `@jokes`  array to find the joke that matches the id that is passed in as an argument in this method.